src/ol_infrastructure/applications/sign_and_verify/__main__.py

```diff
@@ -0,0 +1,228 @@
+from pulumi_aws import acm, ecs, iam, lb, route53, secretsmanager
+
+from ol_infrastructure.lib.ol_types import AWSBase
+
+stack = get_stack()
+stack_name = stack.split('.')[-1]
+namespace = stack.rsplit('.', 1)[0]
+env_suffix = stack_name.lower()
+network_stack = StackReference(f'infrastructure.aws.network.{stack_name}')
+dns_stack = StackReference('infrastructure.aws.dns')
+mitodl_zone_id = dns_stack.require_output('odl_zone_id')
+apps_vpc = network_stack.require_output('applications_vpc')
+aws_config = AWSBase(
+    tags={
+        'OU': 'digital-credentials',
+        'Environment': f'applications-{env_suffix}'
+    }
+)
+
+CONTAINER_PORT = 80
+
+# Create an application load balancer to route traffic to the Fargate service
+
+sign_and_verify_load_balancer = lb.LoadBalancer(
+    f'sign-and-verify-load-balancer-{env_suffix}',
+    name=f'sign-and-verify-load-balancer-{env_suffix}',
+    ip_address_type='dualstack',
+    load_balancer_type='application',
+    enable_http2=True,
+    subnets=apps_vpc['subnet_ids'],
+    security_groups=[apps_vpc['security_groups']['web']],
+    tags=aws_config.merged_tags({'Name': f'sign-and-verify-load-balancer-{env_suffix}'})
+)
+
+sign_and_verify_target_group = lb.TargetGroup(
+    f'sign-and-verify-alb-target-group-{env_suffix}',
+    vpc_id=apps_vpc['id'],
+    target_type='ip',
+    port=CONTAINER_PORT,
+    protocol='HTTP',
+    health_check=lb.TargetGroupHealthCheckArgs(
+        healthy_threshold=2,
+        interval=10,
+        path='/status',
+        port=f'{CONTAINER_PORT}',
+        protocol='HTTP',
+    ),
+    name=f'sign-and-verify-alb-group-{env_suffix}',
+    tags=aws_config.tags,
+)
+
+sign_and_verify_acm_cert = acm.get_certificate(domain='*.odl.mit.edu', most_recent=True, statuses=['ISSUED'])
+
+sign_and_verify_alb_listener = lb.Listener(
+    f'sign-and-verify-alb-listener-{env_suffix}',
+    certificate_arn=sign_and_verify_acm_cert.arn,
+    load_balancer_arn=sign_and_verify_load_balancer.arn,
+    port=443,
+    protocol='HTTPS',
+    default_actions=[
+        lb.ListenerDefaultActionArgs(
+            type='forward',
+            target_group_arn=sign_and_verify_target_group.arn,
+        )
+    ]
+)
+
+# Store the Unlocked DID in AWS secrets manager because it contains private key information
+
+sign_and_verify_config = Config('sign_and_verify')
+unlocked_did_secret = secretsmanager.Secret(
+    f'sign-and-verify-unlocked-did-{env_suffix}',
+    description='Base64 encoded JSON object of the Unlocked DID that specifies the signing keys '
+    'for the digital credentials sign and verify service.',
+    name_prefix=f'sign-and-verify-unlocked-did-{env_suffix}',
+    tags=aws_config.tags
+)
+
+unlocked_did_secret_value = secretsmanager.SecretVersion(
+    f'sign-and-verify-unlocked-did-value-{env_suffix}',
+    secret_id=unlocked_did_secret.id,
+    secret_string=sign_and_verify_config.require_secret('unlocked_did'),  # Base64 encoded JSON object of unlocked DID
```

Comment:
Should this be `unlocked_did_secret`?

Reply:
No, this is referring to the config entry defined here https://github.com/mitodl/ol-infrastructure/pull/19/files#diff-e32743a26673c7e2751688227fe2791404d7dad3865c754decb76be296c97af5R9